packages/portal-proto/src/features/facets/EnumFacet.tsx

```diff
@@ -343,7 +344,7 @@ const EnumFacet: React.FC<FacetCardProps<EnumFacetHooks>> = ({
       </div>
       <div className="h-full">
         {isSuccess && error ? (
-          <div className="m-4 font-content pb-2">{error}</div>
+          <div className="m-4 font-content pb-2">{BAD_DATA}</div>
```

Comment:
should this be `BAD_DATA_MESSAGE`? 

I'm getting some runtime errors when adding the follow_ups.other_clinical_attributes.aids_risk_factors card. 

<img width="1090" alt="Screenshot 2024-07-18 at 4 31 33 PM" src="https://github.com/user-attachments/assets/b4d24429-3ec2-4959-9624-a4e2c2fd3cec">

Reply:
```suggestion
          <div className="m-4 font-content pb-2">{BAD_DATA_MESSAGE}</div>
```